DSharpPlus/Net/Abstractions/Rest/RestGuildPayloads.cs

```diff
@@ -211,6 +211,9 @@ internal sealed class RestGuildRolePayload
 
         [JsonProperty("mentionable", NullValueHandling = NullValueHandling.Ignore)]
         public bool? Mentionable { get; set; }
+
+        [JsonProperty("emoji", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
```suggestion
        [JsonProperty("unicode_emoji", NullValueHandling = NullValueHandling.Ignore)]
```
It is unicode_emoji https://github.com/discord/discord-api-docs/blob/master/docs/resources/Guild.md#json-params-9